app/Main.hs

```diff
@@ -1,8 +1,10 @@
+{-# LANGUAGE NoImplicitPrelude #-}
+
 module Main where
 
-import qualified MyLib (someFunc)
+import Prelude (IO, print)
+
+import Bits
 
 main :: IO ()
-main = do
-  putStrLn "Hello, Haskell!"
-  MyLib.someFunc
+main = print (nor I I)
```

Comment:
``` suggestion
import Prelude (IO, print, (==))

import Bits

main :: IO ()
main = print (nor I I == O) -- should print true
```
Expected output of `(nor I I)` was not obvious, made it explicit. This also demonstrates that boolean logic can be applied to our custom `Bit` type.

Reply:
Good idea. Thanks.